### DISCUSSION TITLE
Welcome to Node-RED-HA-Octopus-Agile-JSONata Discussions!

### DISCUSSION THREAD

**Discussion by oatybiscuit** [oatybiscuit/Node-RED-HA-Octopus-Agile-JSONata#1] [Announcements]
## 👋 Welcome!

 Discussions is a place to connect with other members of our community.
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by ando2040** [comment]
Nice work setting up a git page - as always Geoff! I'll take some deep breaths and install the updated version in a test instance soon. In the meantime even though you didn't intend it to be used for real automations, the original is working nicely most of the time.

**Comment by raldred** [comment]
Just wanted to pass on my thanks for all your efforts with this project @oatybiscuit. I'm sure others like I am, are incredibly grateful for you sharing you work.
